spec/factories/articles.rb

```diff
@@ -5,5 +5,8 @@
     title { 'this is title' }
     body { 'this is body, lorem ipsum.' * 20 }
     category { 'sport' }
+    trait :with_image do
```

Comment:
what is trait?

Reply:
I think that's what you have to write when it `has_one_attached`.

Reply:
Since it is not a db column, it needs some special care